src/wix/WixToolset.Core.Burn/Bundles/ProcessMsiPackageCommand.cs

```diff
@@ -397,6 +399,8 @@ private void ImportExternalCabinetAsPayloads(Database db, WixBundlePayloadSymbol
                                     Packaging = packagePayload.Packaging,
                                     ParentPackagePayloadRef = packagePayload.Id.Id,
                                 });
+
+                                this.CheckIfWindowsInstallerFileTooLarge(this.Facade.PackageSymbol.SourceLineNumbers, payloadSourceFile, "cabinet");
```

Comment:
Doesn't this check if the cabinet files are too large in the bundle?

Reply:
Yes, for `MsiPackage` files. But another check is needed for containers created by  https://github.com/wixtoolset/wix4/blob/73abf1db11a2ec38937682563d382a5c67fe2c40/src/wix/WixToolset.Core.Burn/Bundles/CreateContainerCommand.cs, right?

Reply:
Not as long as Burn can handle the files. My understanding is that MSI can't handle cabinet files over 2GB not Burn.

Reply:
I have not tested it. I guess it makes sense that if wixnative can create the cab then Burn should be able to read it, but I'm not 100% sure on that.